svd/patches/_interrupts.yaml

```diff
@@ -0,0 +1,490 @@
+# Draft attempt at adding interrupt mappings to peripherals.
+
+# Some peripherals are not publicly documented, but we can infer their
+# existence from the ESP-IDF sources and object files. However, naming
+# is inconsistent between various parts of the ESP-IDF source and the
+# ESP32 Technical Reference.
+#
+# Therefore the strategy used here is to take the name of the peripheral
+# from the interrupt name obtained by `idf2svd` from the ESP-IDF source,
+# except where it is obvious that the interrupt belongs to a peripheral
+# which has already defined by another name.
+#
+# In most cases interrupts map cleaning using this strategy, except some
+# peripherals need to be invented below.
+
+# Open issues:
+#   * should SLC be split into SLC0 and SLC1?
+#   * the comments on the two I2C controllers seem reversed
+#   * unclear where TIMER1 and TIMER2 should live
```

Comment:
TIMER1 & TIMER2 are the xtensa built in peripherals right? So probably xtensa crate (currently empty in esp-rs)

Reply:
No, they aren't. These are two timers outside of the CPU (i.e., peripherals).